server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -485,6 +485,12 @@ public void run() {
       }
     }
     log.debug("Using {} to advertise monitor location in ZooKeeper", advertiseHost);
+    try {
+      monitorLock.replaceLockData(
```

Comment:
it looks like this utility is already called a few lines later so you could probably refactor this to call it one time and use that result. https://github.com/apache/accumulo/blob/46ed92329cf83911c5b43f348658941e0979f3f9/server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java#L495-L496

Reply:
I should clarify specially i'm referring to using `HostAndPort.fromParts(advertiseHost, livePort)`